pkg/agent/openflow/pipeline.go

```diff
@@ -41,38 +41,127 @@ import (
-	ServiceLBTable               = binding.NewOFTable(41, "ServiceLB")
-	EndpointDNATTable            = binding.NewOFTable(42, "EndpointDNAT")
-	AntreaPolicyEgressRuleTable  = binding.NewOFTable(45, "AntreaPolicyEgressRule")
-	DefaultTierEgressRuleTable   = binding.NewOFTable(49, "DefaultTierEgressRule")
-	EgressRuleTable              = binding.NewOFTable(50, "EgressRule")
-	EgressDefaultTable           = binding.NewOFTable(60, "EgressDefaultRule")
-	EgressMetricTable            = binding.NewOFTable(61, "EgressMetric")
-	L3ForwardingTable            = binding.NewOFTable(70, "L3Forwarding")
-	SNATTable                    = binding.NewOFTable(71, "SNAT")
-	L3DecTTLTable                = binding.NewOFTable(72, "IPTTLDec")
-	L2ForwardingCalcTable        = binding.NewOFTable(80, "L2Forwarding")
-	AntreaPolicyIngressRuleTable = binding.NewOFTable(85, "AntreaPolicyIngressRule")
-	DefaultTierIngressRuleTable  = binding.NewOFTable(89, "DefaultTierIngressRule")
-	IngressRuleTable             = binding.NewOFTable(90, "IngressRule")
-	IngressDefaultTable          = binding.NewOFTable(100, "IngressDefaultRule")
-	IngressMetricTable           = binding.NewOFTable(101, "IngressMetric")
-	ConntrackCommitTable         = binding.NewOFTable(105, "ConntrackCommit")
-	ServiceConntrackCommitTable  = binding.NewOFTable(106, "ServiceConntrackCommit")
-	HairpinSNATTable             = binding.NewOFTable(108, "HairpinSNAT")
-	L2ForwardingOutTable         = binding.NewOFTable(110, "Output")
+	// PipelineClassifierTable is table 0. Packets are forwarded to different pipelines in this table. Note that, the value
+	// of stage and pipeline arguments for this table is dummy. This table is always realized to OVS.
+	PipelineClassifierTable = newTable("PipelineClassifier", binding.StageAll, binding.PipelineAll)
+
+	//      _   _   _             _   _               _
+	//     / \ | |_| |_ ___ _ __ | |_(_) ___  _ __   | |
+	//    / _ \| __| __/ _ \ '_ \| __| |/ _ \| '_ \  | |
+	//   / ___ \ |_| ||  __/ | | | |_| | (_) | | | | |_|
+	//  /_/   \_\__|\__\___|_| |_|\__|_|\___/|_| |_| (_)
+	//
+	// Before adding a new table here, please read the following instruction carefully.
+	// - Double confirm the necessity of adding a new table, and consider how to reuse an existing table to implement
+	//   alternative functionality.
+	// - Choose a good name. A good name is very helpful for users to understand the function of the table.
+	// - Choose a stage. All tables in a stage are intended to implement a functional module in pipelines. All stages are
+	//   defined in file pkg/ovs/openflow/interfaces.go. If you want to add a new stage, please discuss with maintainers
+	//   or developers of Antrea.
+	// - Choose a target pipeline. All pipelines are defined in file pkg/ovs/openflow/interfaces.go. If you want to add a
+	//   new pipeline, please discuss with maintainers or developers of Antrea.
+	// - The most important thing is to choose the right place to declare the new table. For example:
```

Comment:
I am not sure this is "the most important thing". Maybe say: "Decide where to add the new table in the pipeline. The order table declaration decides the order of tables in the stage".